.github/workflows/windows.yml

```diff
@@ -0,0 +1,56 @@
+name: windows
+
+on:
+  push:
+    paths-ignore:
+    - '.github/workflows/**'
+    - '!.github/workflows/windows.yml'
+  pull_request:
+    paths-ignore:
+    - '.github/workflows/**'
+    - '!.github/workflows/windows.yml'

```

Comment:
TIL paths-ignore, thank you :100: 

Out of curiosity: shouldn't `EnterDevShell.ps1` be something provided by the WIndows/MSVC SDK/DDKs or alternatively the Windows container/image provided by Github workflow?

The CI coverage looks great, thanks o/